config/locales/mailers/en.yml

```diff
@@ -83,9 +83,11 @@ en:
         support: The AO3 Support team
   roles:
     anonymous_creator: Anonymous Creator
-    guest_commenter_name_html: "%{name} %{role}"
-    guest_commenter_name_text: "%{name} %{role}"
+    commenter_name_html: "%{name} %{role}"
+    commenter_name_text: "%{name} %{role}"
```

Comment:
Can we rename the `%{role}` variable `%{role_with_parens}`? Translation finds it helpful for the variable name to closely correspond with the key for the text the variable represents.